tailwind.config.js

```diff
@@ -66,19 +66,58 @@ module.exports = {
       2: '2px',
     },
     fontSize: {
-      // Body
-      sm: '0.80rem',
-      base: '1.00rem',
+      // Deprecated
       lg: '1.12rem',
       lgr: '1.43rem',
 
       // Headings, where h[n] => [7-n]xl
-      '1xl': '1.12rem',
-      '2xl': '1.25rem',
-      '3xl': '1.43rem',
-      '4xl': '1.75rem',
-      '5xl': '2.25rem',
-      '6xl': '3.56rem',
+      '6xl': [
+        '2.2857142857rem', // 32px
+        '1.3', // 130%
```

Comment:
What does the second number in the array do? I couldn't find information about it the tailwind configuration docs.

Edit: Oh, is it line-height or something?

Reply:
yep! line height.

Reply:
For posterity, here is the [relevant documentation](https://tailwindcss.com/docs/font-size#providing-a-default-line-height).